keyboards/decent/tkl/tkl.c

```diff
@@ -0,0 +1,62 @@
+// Copyright 2024 B. Le Roy
+// SPDX-License-Identifier: GPL-2.0-or-later
+
```

Comment:
I really suggest you test that your code compiles prior to requesting reviews.
```suggestion
#include "quantum.h"

```